### PR TITLE
feat: remove approval process for DC prod integrations

### PR DESCRIPTION
**Pull request by NithinKuruba**

### CODE REVIEW

File: lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+
+export const name = '2024.09.16T00.00.00.remove-vc-approved';
+
+export const up = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().removeColumn('requests', 'digital_credential_approved');
```
Comment (arcshiftsolutions) [inline lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts:6]: Still required?

File: lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+};
+
+export const down = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn('requests', 'digital_credential_approved', {
```
Comment (arcshiftsolutions) [inline lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts:10]: Still required?

**Review by arcshiftsolutions — APPROVED** [review]

File: lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts (hunk 1 above)
Comment (NithinKuruba) [reply lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts:6]: yes its required for sequelize to remove the column

File: lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts (hunk 2 above)
Comment (NithinKuruba) [reply lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts:10]: yes its required for sequelize to remove the column

**Pull request merged**